shared/chat/conversation/messages/popup.desktop.js

```diff
@@ -15,19 +16,22 @@ const stylePopup = {
 export const TextPopupMenu = ({message, onShowEditor, onDeleteMessage, onHidden, style, you}: TextProps) => {
   let items = []
   if (message.author === you) {
-    items = [
-      {onClick: () => onShowEditor(message), title: 'Edit'},
-      {
-        danger: true,
-        onClick: () => onDeleteMessage(message),
-        subTitle: 'Deletes for everyone',
-        title: 'Delete',
-      },
-    ]
-
     if (!message.senderDeviceRevokedAt) {
-      items.unshift('Divider')
+      items.push('Divider')
+    }
+
+    if (textMessageEditable(message)) {
+      items.push({onClick: () => onShowEditor(message), title: 'Edit'})
+    } else {
+      items.push({disabled: true, title: 'Edit'})
```

Comment:
setting disabled isn't enough (!), must also remove onClick

Reply:
yeah we could update the component do this this for us